postgres_mixin/alerts/postgres.libsonnet

```
@@ -1,22 +1,22 @@
 {
-  prometheusAlerts+:: {
+  prometheusAlerts+: {
```

```suggestion
  prometheusAlerts+:: {
```

This should be `+::` by monitoring-mixin convention otherwise it woudn't load in some cases (like grizzly).